app.js

```diff
@@ -1,5 +1,6 @@
 const burger =  document.querySelector('#burger');
 const menu =  document.querySelector('#menu');
+
 burger.addEventListener('click',()=> {
     menu.classList.toggle('disp')
```

Comment:
[Говноссылка](https://andchuiko.github.io/restaurant/)

Было бы лучше использовать читаемые названия классов, так как `disp` не совсем понятно...
```suggestion
    menu.classList.toggle('show')
```